Martium.GuessTheRiddleGame/Program.cs

```diff
@@ -22,16 +22,15 @@ internal class Program
 
         private static int SingleGuessPoints = 1;
         private static int GuessLimit = 3;
+        private static int GameNumber = 1;
```

Comment:
instead of creating field, you can create variable in `StartMainGame` method and pass parameter to ShowGameNumber method (this field does not give any value as it is used only in one methods logic

Reply:
i use it on two methods "GameNumber" variable thats why i use as field